modules/http/http-parser.c

```diff
@@ -48,7 +48,10 @@ static CfgLexerKeyword http_keywords[] =
   { "accept_redirects", KW_ACCEPT_REDIRECTS },
   { "timeout",          KW_TIMEOUT },
   { "tls",              KW_TLS },
-  { "flush_bytes",      KW_FLUSH_BYTES },
+  { "flush_bytes",      KW_BATCH_BYTES, KWS_OBSOLETE, "The flush-bytes option is deprecated. Use batch-bytes instead." },
+  { "batch_bytes",      KW_BATCH_BYTES },
+  { "flush_lines",      KW_BATCH_LINES, KWS_OBSOLETE, "The flush-lines option is deprecated. Use batch-lines instead."},
+  { "flush_timeout",    KW_BATCH_TIMEOUT, KWS_OBSOLETE, "The flush-timeout option is deprecated. Use batch-timeout instead."},
   { "body_prefix",      KW_BODY_PREFIX },
```

Comment:
I think we can the batching related keywords as they are already defined in cfg-parser.c, which should work here too.